cmd/cmd.go

```diff
@@ -686,7 +695,7 @@ func generateInteractive(cmd *cobra.Command, model string) error {
 		}
 
 		if len(line) > 0 && line[0] != '/' {
-			if err := generate(cmd, model, line, wordWrap); err != nil {
+			if err := generate(cmd, model, line, wordWrap, format); err != nil {
```

Comment:
Not super important now, but we probably should turn this into a generateParams struct at some point to cut down on the stuff we're passing.